doc/source/1-TUTORIALS/3-CONTRIBUTE-TO-DDISASM/1-Build-Documentation.rst

```diff
@@ -0,0 +1,110 @@
+Contribute to ddisasm documentation
+===================================
+
+Introduction
+------------
+
+Disassm uses the template of the 4 types of documentations including:
+1-tutorials (to get started)
+2-how to (real life issues solving)
+3-references (the autogenerated api)
+4-explaination (for the concepts)
```

Comment:
Since we don't have content for 4, I would leave it out for now.

Reply:
I was going to provide a "four types of documentation templates". This is about the context of the project. 

- What can we do with reverse engineering in general. Not specifically any ctf. Some laws about reverse engineering to use it safely.
- alternatives such as radare2, ghidra and IDA pro disassemblers. I will tell all the bugs in r2 as well as the lack of ability to reconstruct a full binary in ghidra and IDA pro. I will also tell that ddisasm is slow and consumes ram.


As you mentionned. I have not wrote this yet. Can I write it or do I remove the chapter?